_posts/2020-03-09-o-openid-connect-slow-kilka.md

```diff
@@ -0,0 +1,106 @@
+---
+layout:    post
+title:     "O OpenID Connect słów kilka"
+published: true
+date:      2020-03-09 08:00:00 +0100
+author:    mhoja
+tags:
+    - openid connect
+    - oauth 2
+    - keycloak
+    - uwierzytelnianie użytkownika
+---
+
+W nawiązaniu do mojego poprzedniego wpisu pt.:  
+[**"Keycloak - uwierzytelnianie i autoryzacja użytkownika w aplikacji Angular/Spring Boot"** (link](https://blog.consdata.tech/2020/02/01/keycloak-uwierzytelnianie-autoryzacja-springboot-angular.html)),  
+chciałbym krótko opisać standard [OpenID Connect](https://openid.net/connect/), który został wykorzystany podczas logowania do aplikacji przy użyciu serwera uwierzytelniania [Keycloak](https://www.keycloak.org/).
+
+## Wstęp
+
+Najpopularniejszymi standardami wykorzystywanymi do uwierzytelniania/autoryzacji są `OAuth 2.0`, `OpenID Connect` oraz `SAML`.  
+O OAuth 2.0 zostało już napisanych wiele artykułów, których nie ma sensu powielać. Jednak aby przedstawić OpenID Connect, musiałbym opisać [OAuth 2.0](https://oauth.net/2/) oraz [JWT](https://jwt.io/).
```

Comment:
"Jednak" może rozpoczynać zdanie i nie jest to błąd, ale zazwyczaj lepiej brzmi w środku zdania, może tutaj by tak zrobić? "Aby jednak przedstawić OpenID [...]"

Reply:
akurat jak to czytam, to tutaj bym zostawił jak jest